README_.md

```diff
@@ -0,0 +1,177 @@
+The project is based on [MLIR](https://mlir.llvm.org/) and highly related with
+[mlir-hlo](https://github.com/tensorflow/mlir-hlo) project.
+
+Refer to [our website](https://alibaba.github.io/BladeDISC/) for more
+information, including the setup tutorial, developer guide, demo examples and
+documents for developers.
+
+## 1.1 Features and Roadmap
+
+### 1.1.1 Frontend Framework Support Matrix
+
+|           | TensorFlow [1] | Pytorch [2]  |
+|---------- | -------------- | ------------ |
+| Inference |    Yes         |    Yes       |
+|  Training |    Yes [3]     |  Ongoing     |
+
+[1] TensorFlow ver1.12, 1.15, 2.4 & 2.5 are supported and fully verified. For
+other versions some slight works on adaptation might be needed.
+
+[2] Torch 1.6.0 <= version < 1.9.0 has been fully verified.
+
+[3] Although supported, there's much room for imporvement on Op coverage for
+training workloads.
+
+### 1.1.2 Backend Support Matrix
+
+|    | Memory Intensive Part | Compute Intensive Part | End-to-End Usability |
+|----------- | ------------- | ---------------------- | -------------------- |
+| Nvidia GPU |    Yes        |    Yes                 |    Yes               |
+| AMD GPU    |  Ongoing      |  Ongoing               |     No               |
+| Hygon DCU  |    Yes        |    Yes                 |    Yes               |
+|  X86       |    Yes        |  Not open-sourced yet [1]  |     No           |
+
+[1] The compute intensive part of the X86 backend is already supported on the
+internal version. The code decoupling is ongoing and will be open-sourced soon,
+same for the end-to-end usability.
+
+### 1.1.3 Deployment Solutions
+
+* Plugin Mode - BladeDISC works as a pluggin of TensorFlow or PyTorch. Only the
+  supported Ops are clustered and compiled, and the unsupported ones will be
+  executed by the the host runtime. We recommand this mode to most of the users
+  for its transparency and ease of use. 
+
+* Standalone Mode - In Standalone mode, the input workload will be compiled into
+  a binary that can be executed by it self, aka, does not rely on a TensorFlow
+  or LibTorch runtime. In this mode all ops must be supported. 
+
+## 1.2 Numbers of Typical Workloads
+
+By evaluating BladeDISC using a set of typical machine learning workloads for
+production purpose, DISC shows up to 3x speedup comparing with TensorFlow/PyTorch.
+
+<div align="centor">
+<img src=/docs/numbers.png width=60% />
+</div>
+
+### 1.2.1 Advantage in Dynamic Shape Workloads
+
+Specifically, for the BERT large inference on T4 we provide in the
+[examples](/docs/example/tensorflow/BERT_inference.md), static compiler
+optimization (XLA) shows severe performance degradation due to its compilation
+overhead, while DISC shows a 1.75x speedup.
+
+| TensorFlow  |    XLA    |    DISC    |
+|-------------|-----------|------------|
+|   1.78 s    |   41.69s  |    1.02s   |
+|   1X        |           |    1.75X   |
+
+# 2 QuickStart
```

Comment:
Maybe it's too long in the README on the project root. I prefer a **quick view** section that introduces the simple BladeDISC API for Pytorch and TensorFlow wrapper, that users can force on **WHAT** is BladeDISC, and we can reference **HOW** to use it step by step in a separate file.

The `quick view` section can use some pseudocode just like the following:

```
# enable BladeDISC on TensorFlow program
import tensorflow_blade_disc as disc
disc.enable()

# construct TensorFlow Graph and run it
g = tf.Graph()
with g.as_default():
    ...
    with tf.session as sess:
        sess.run(...)
```

Reply:
agree, done